apps/molgenis-viz/src/components/viz/ColumnChartEmx.vue

```diff
@@ -51,6 +51,11 @@ const emit = defineEmits<{
   (e: "viz-data-clicked", row: object): void;
 }>();
 
+const graphqlEndpoint = computed<string>(() => {
```

Comment:
to stop this code duplication might this be something we could make a global service?

I mean, something like this?
https://vuejs.org/api/application.html#app-config-globalproperties